jvb/src/test/kotlin/org/jitsi/videobridge/ConferenceTest.kt

```diff
@@ -17,53 +17,33 @@ package org.jitsi.videobridge
 
 import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.matchers.shouldBe
-import io.mockk.every
-import io.mockk.mockk
-import io.mockk.mockkStatic
 import org.jitsi.ConfigTest
-import org.jitsi.videobridge.octo.OctoRelayService
-import org.jitsi.videobridge.octo.OctoRelayServiceProvider
-import org.jitsi.videobridge.octo.singleton
+import org.jitsi.videobridge.octo.singleton as octoRelayServiceProvider
 import org.json.simple.JSONObject
 import org.json.simple.parser.JSONParser
 import org.jxmpp.jid.impl.JidCreate
-import org.osgi.framework.BundleContext
 
 /**
  * This is a high-level test for [Conference] and related functionality.
  */
 class ConferenceTest : ConfigTest() {
-    private val octoRelayService = OctoRelayService()
-    private val octoRelaySupplier: OctoRelayServiceProvider = mockk() {
-        every { get() } returns octoRelayService
-    }
-
-    private val bundleContext = mockk<BundleContext> {
-        every { registerService(any() as String, any(), any()) } returns null
-    }
-    private val videobridge = mockk<Videobridge> {
-        every { bundleContext } returns this@ConferenceTest.bundleContext
-    }
-
-    init {
-        mockkStatic("org.jitsi.videobridge.octo.OctoRelayServiceProviderKt")
-        every { singleton() } returns octoRelaySupplier
-    }
+    private val videobridge = Videobridge()
```

Comment:
I've got mixed feelings on using a real `Videobridge` here as it will make the test more brittle.  I know there's a counter argument that the test may be less accurate with a mock version, but I think it's better to mock the dependencies so they are better controlled for the test.